tsl/src/continuous_aggs/invalidation.c

```diff
@@ -302,53 +298,46 @@ invalidation_hyper_log_add_entry(int32 hyper_id, int64 start, int64 end)
 /*
  * Invalidate one or more continuous aggregates.
  *
- * Add an invalidation in the given range. The invalidation is added either to
- * the hypertable invalidation log or the continuous aggregate invalidation
- * log depending on the type of the given hypertable. If the hypertable is a
- * "raw" hypertable (i.e., one that has one or more continous aggregates), the
- * entry is added to the hypertable invalidation log and will invalidate all
- * the associated continuous aggregates. If the hypertable is instead an
- * materialized hypertable, the entry is added to the cagg invalidation log
- * and only invalidates the continuous aggregate owning that materialized
- * hypertable.
+ * Add an invalidation in the given range to the continuous aggregate invalidation
+ * log.
+ *
+ * The entry is added to the hypertable invalidation log and will invalidate all
+ * the associated continuous aggregates.
  */
 void
-invalidation_add_entry(const Hypertable *ht, ContinuousAggHypertableStatus caggstatus,
-					   int32 entry_id, int64 start, int64 end)
+invalidate_continuous_agg(const Hypertable *ht, int64 start, int64 end)
```

Comment:
The name `invalidate_continuous_agg()` is confusing.

The terminology is:
- "hypertable invalidation log"
- "continuous aggregate invalidation log" or "materialization invalidation log"

Reply:
This rename is confusing/misleading. The function adds an entry to the invalidation log. 
In general, I find the refactoring/renaming changes pretty confusing.